dask_jobqueue/htcondor.py

```diff
@@ -220,3 +194,34 @@ def quote_environment(env):
         entries.append("%s=%s" % (k, qv))
 
     return " ".join(entries)
+
+
+class HTCondorCluster(JobQueueCluster):
+    __doc__ = """ Launch Dask on an HTCondor cluster with a shared file system
+
+    Parameters
+    ----------
+    disk : str
+        Total amount of disk per job
+    job_extra : dict
+        Extra submit file attributes for the job
+    {job}
+    {cluster}
+
+    Examples
+    --------
+    >>> from dask_jobqueue.htcondor import HTCondorCluster
+    >>> cluster = HTCondorCluster(cores=24, memory="4GB", disk="4GB")
+    >>> cluster.scale(10)
+
+    >>> from dask.distributed import Client
+    >>> client = Client(cluster)
+
+    This also works with adaptive clusters.  This automatically launches and kill workers based on load.
+
+    >>> cluster.adapt(minimum=5)
```

Comment:
I think using `minimum` is very rarely useful. Probably the most useful example is to use `maximum_jobs`

Reply:
Fixed